classes/php.php

```diff
@@ -41,6 +41,10 @@ public function setBinaryPath($phpPath = null)
 			}
 		}
 
+		if (defined('HHVM_VERSION')) {
```

Comment:
I might want to move this here, https://github.com/atoum/atoum/blob/master/classes/test/engines/concurrent.php#L178-L181.